projects/1.18/assets/blockcarpentry/blockcarpentry/lang/zh_cn.json

```
@@ -1,37 +1,105 @@
-  "block.blockcarpentry.illusion_chest": "幻象箱子",
-  "block.blockcarpentry.illusion_ladder": "幻象梯子",
-  "block.blockcarpentry.illusion_wall": "幻象墙",
-  "block.blockcarpentry.illusion_bed": "幻象床",
-  "block.blockcarpentry.illusion_fence": "幻象栅栏",
-  "block.blockcarpentry.illusion_trapdoor": "幻象活板门",
-  "block.blockcarpentry.illusion_door": "幻象门",
-  "block.blockcarpentry.illusion_pressure_plate": "幻象压板",
-  "block.blockcarpentry.illusion_stairs": "幻象楼梯",
-  "block.blockcarpentry.illusion_button": "幻象按钮",
-  "block.blockcarpentry.illusion_slab": "幻象台阶",
-  "block.blockcarpentry.illusion_block": "幻象方块",
-  "block.blockcarpentry.frame_edged_slope": "边缘框架坡度（WIP）",
-  "block.blockcarpentry.frame_slope": "框架坡度（WIP）",
-  "block.blockcarpentry.frame_fence_gate": "框架栅栏门",
-  "block.blockcarpentry.frame_chest": "框架箱子",
-  "block.blockcarpentry.frame_ladder": "框架梯子",
-  "block.blockcarpentry.frame_wall": "框架墙",
-  "block.blockcarpentry.frame_bed": "框架床",
-  "block.blockcarpentry.frame_fence": "框架栅栏",
-  "block.blockcarpentry.frame_trapdoor": "框架活板门",
-  "block.blockcarpentry.frame_door": "框架门",
-  "block.blockcarpentry.frame_pressure_plate": "木制框架压板",
-  "block.blockcarpentry.frame_stairs": "框架楼梯",
+  "block.blockcarpentry.frameblock": "框架",
+  "block.blockcarpentry.frame_slab": "框架台阶",
+  "block.blockcarpentry.falling_frameblock": "框架（受重力影响 - WIP）",
   "block.blockcarpentry.frame_button": "木制框架按钮",
-  "block.blockcarpentry.falling_frameblock": "框架（受重力影响-WIP）",
-  "block.blockcarpentry.frame_slab": "框架板",
-  "block.blockcarpentry.frameblock": "框架"
+  "block.blockcarpentry.frame_stairs": "框架楼梯",
+  "block.blockcarpentry.frame_pressure_plate": "木制框架压力板",
+  "block.blockcarpentry.frame_door": "框架门",
+  "block.blockcarpentry.frame_trapdoor": "框架活板门",
+  "block.blockcarpentry.frame_fence": "框架栅栏",
+  "block.blockcarpentry.frame_bed": "框架床",
+  "block.blockcarpentry.frame_wall": "框架墙",
+  "block.blockcarpentry.frame_ladder": "框架梯子",
+  "block.blockcarpentry.frame_chest": "框架储物箱",
+  "block.blockcarpentry.frame_fence_gate": "框架栅栏门",
+  "block.blockcarpentry.frame_carpet": "框架地毯",
+  "block.blockcarpentry.frame_pane": "框架玻璃板",
+  "block.blockcarpentry.frame_daylight_detector": "框架日光传感器",
+  "block.blockcarpentry.frame_layered_block": "框架分层方块",
+  "block.blockcarpentry.frame_slope": "框架斜坡",
+  "block.blockcarpentry.frame_edged_slope": "边缘框架斜坡",
+  "block.blockcarpentry.illusion_block": "幻象方块",
+  "block.blockcarpentry.illusion_slab": "幻象台阶",
+  "block.blockcarpentry.illusion_button": "幻象按钮",
+  "block.blockcarpentry.illusion_stairs": "幻象楼梯",
+  "block.blockcarpentry.illusion_pressure_plate": "幻象压力板",
+  "block.blockcarpentry.illusion_door": "幻象门",
+  "block.blockcarpentry.illusion_trapdoor": "幻象活板门",
+  "block.blockcarpentry.illusion_fence": "幻象栅栏",
+  "block.blockcarpentry.illusion_bed": "幻象床",
+  "block.blockcarpentry.illusion_wall": "幻象墙",
+  "block.blockcarpentry.illusion_ladder": "幻象梯子",
+  "block.blockcarpentry.illusion_chest": "幻象储物箱",
```

```suggestion
  "block.blockcarpentry.illusion_chest": "幻象箱子",
```
这两个要看游戏内是不是对应原版箱子，不是就不改